doc/main/hubs/primehub.rst

```diff
@@ -1,4 +1,4 @@
-.. pybricks-requirements:: primehub
+.. pybricks-requirements:: essentialhub
```

Comment:
Looks like this got renamed on the wrong page.

Reply:
Oh my 🤦‍♂️ . Reverted.